extras/paid-books.md

```diff
@@ -91,6 +91,11 @@ Name | Author | ISBN
 ### Cloud Computing
 
 ### Math (Linear Algebra)
+Name | Author | ISBN
+:-- | :--: | :--:
+[Linear Algebra Done Right third edition](http://www.amazon.in/Linear-Algebra-Right-Undergraduate-Mathematics/dp/0387982582) | Sheldon Axler | 0387982582
+
```

Comment:
Actually if you look at the preview, the second book isn't part of the table. 
![screen shot 2016-10-13 at 9 14 58 am](https://cloud.githubusercontent.com/assets/2111074/19357228/95f6f1c4-9125-11e6-8bbe-218621a872ae.png)

If you remove that extra line, it should show up correctly. Hope that helps!